src/tfrobot/tfrobot_commands_flags.md

```diff
@@ -0,0 +1,59 @@
+<h1> Commands and Flags </h1>
+
+<h2>Table of Contents</h2>
+
+- [Introduction](#introduction)
+- [Commands](#commands)
+- [Subcommands](#subcommands)
+- [Flags](#flags)
+
+***
+
+## Introduction
+
+We present the various commands, subcommands and flags available with TFROBOT.
+
+
+## Commands
+
+You can run the command `tfrobot help` at any time to access the help section. This will also display the available commands.
+
+| Command    | Description                                                |
+| ---------- | ---------------------------------------------------------- |
+| completion     | Generate the autocompletion script for the specified shell                        |
+| help       | Help about any command                                     |
+| version    | Get latest build tag                                       |
+
+You can also use the following flags: `-c` (`--config`) and `-h` (`--help`).
```

Comment:
This part feels repeated; you already mentioned them down on the flag table.

Reply:
Done. If I understood correctly, I removed the line `You can also use the following flags: `-c` (`--config`) and `-h` (`--help`).`